WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/UseCases/RefundCreationUseCase.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+import Yosemite
+
+/// Creates a `Refund` object ready to be used  on `RefundStore.createRefund` action
+///
+struct RefundCreationUseCase {
+
+    /// Total amount to be refunded.
+    ///
+    let amount: String
```

Comment:
just wondering, can `amount` be derived from `items`? or this total amount could include other information that this use case doesn't know?

Reply:
`amount` could be derived from `items` + `shipping` but the screen that will call this method already has the amount calculated and displayed to the user, that's why I decided to just pass it on. But will consider it again once I add the shipping support!